### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by Rachael-E**
### Description

Readme updates and where appropriate, associated minor code updates.

### CODE REVIEW

**Comment by Rachael-E** [comment]
@sclaridge please could you take a look? thanks!

File: utility_network/display-utility-associations/README.md [hunk 1]
```diff
@@ -36,6 +36,10 @@ Pan and zoom around the map. Observe graphics that show utility associations bet
 
 The [Naperville electrical](https://sampleserver7.arcgisonline.com/server/rest/services/UtilityNetwork/NapervilleElectric/FeatureServer) network feature service, hosted on ArcGIS Online, contains a utility network used to display the utility associations in this sample.
 
+## Additional information
+
+Using utility network on ArcGIS Enterprise 10.8 requires an ArcGIS Enterprise member account licensed with the [Utility Network user type extension](https://enterprise.arcgis.com/en/portal/latest/administer/windows/license-user-type-extensions.htm#ESRI_SECTION1_41D78AD9691B42E0A8C227C113C0C0BF). Please refer to the [utility network services documentation](https://enterprise.arcgis.com/en/server/latest/publish-services/windows/utility-network-services.htm).
```
Comment (sclaridge) [inline utility_network/display-utility-associations/README.md:41]:
```suggestion
Using a utility network on ArcGIS Enterprise 10.8 requires an ArcGIS Enterprise member account licensed with the [Utility Network user type extension](https://enterprise.arcgis.com/en/portal/latest/administer/windows/license-user-type-extensions.htm#ESRI_SECTION1_41D78AD9691B42E0A8C227C113C0C0BF). Please refer to the [utility network services documentation](https://enterprise.arcgis.com/en/server/latest/publish-services/windows/utility-network-services.htm).
```

File: editing/edit-with-branch-versioning/README.md [hunk 2]
```diff
@@ -44,13 +44,18 @@ The feature layer used in this sample is [Damage to commercial buildings](https:
...
+* Passowrd: S7#i2LWmYH75
+
 The name of the version must meet the following criteria:
 
 1. Must not exceed 62 characters
 2. Must not include: Period (.), Semicolon (;), Single quotation mark ('), Double quotation mark (")
 3. Must not include a space for the first character
 
-**Note**: the version name will have the username and a period (.) prepended to it. E.g "editor01.MyNewUniqueVersionName"
+**Note**: the version name will have the username and a period (.) prepended to it e.g "editor01.MyNewUniqueVersionName"
```
Comment (sclaridge) [inline editing/edit-with-branch-versioning/README.md]:
```suggestion
**Note**: the version name will have the username and a period (.) prepended to it e.g "editor01.MyNewUniqueVersionName".
```

**Review by sclaridge — APPROVED** [review]
Looks great! I added a few minor suggestions, but I'll leave those to your discretion.

**Pull request merged**